setup.py

```diff
@@ -72,7 +74,9 @@
     include_package_data=True,
     install_requires=requires,
     extras_require={
-      "debug": ["rpdb"]
+        "debug": ["rpdb"],
+        # option to install a matched pair of inmanta-core and pytest-inmanta-extensions
+        "pytest-inmanta-extensions": [f"pytest-inmanta-extensions~={version}.0.dev"],
```

Comment:
Have you tested this? Why `.0.dev` and not just `.dev`? Does this also allow non-dev versions? Does the release tool need to update this on release?

Reply:
this translates into 

`>={version}.0.dev =={version}.*`

I think it it allows all dev and releases (and posts) that start with `version` 

I would not update in release tool, once released, {version} will be the highest number matching this expression

Reply:
https://www.python.org/dev/peps/pep-0440/#compatible-release, bottom of this section, I have not tested this.

Reply:
I see, thanks.